lib/cylc/gui/app_gcylc.py

```diff
@@ -533,8 +533,10 @@ def __init__(self, suite, db, owner, host, port, pyro_timeout,
 
         setup_icons()
 
-        self.view_layout_horizontal = False
-
+        if gcfg.get(['initial orientation']) == "side-by-side":
```

Comment:
Agreed, if not doing the above initial side-by-side views type config item then this would be better.